package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "tukutoi-cp-directory-integration",
+	"version": "0.1.0",
```

Comment:
Pleas apologise my ignorance - should this not state the current Plugin version, so translations are updated if the version changes? I saw the same in the -lock file

Thanks!

Reply:
Feel free to update that. But it doesn't matter for the localization and generating of the .pot file. That's versioning of the build process.